Bowling.java

```diff
@@ -2,91 +2,91 @@
 
 public class Bowling {
 
-  private Scoreboard scoreboard;
+	private Scoreboard scoreboard;
```

Comment:
Ah - now you get to see why there are "religious wars" over spacing.  Looks like you changed from spaces to tabs or the number of spaces used to represent a tag.  Having format changes mixed in with content changes makes it hard to see what really changed.  If you add "?w=1" to the end of the github url it will ignore whitespace changes and make it more readable.

We should probably choose a spacing though, so we don't keep stomping on each other.  I personally prefer tabs, but I'm willing to do spaces - in which case I prefer 2 (but am willing to do 4)...:)